pkg/upgrade/base/consts.go

```diff
@@ -39,6 +39,8 @@ const (
 	// (used for orphaned pods)
 	// Setting this label will trigger setting upgrade state to upgrade-required
 	UpgradeRequestedAnnotationKeyFmt = "nvidia.com/%s-driver-upgrade-requested"
+	// UpgradeRequestorModeAnnotationKeyFmt
+	UpgradeRequestorModeAnnotationKeyFmt = "nvidia.com/%s-driver-upgrade-requestor-mode"
```

Comment:
Can we add a short README section on the difference between the `legacy` and `requestor` modes?